Issue with go 1.14.0, Helm, 3.1.1

Hi,

I am using Helm as a package from within my application (https://github.com/shipyard-run/shipyard/blob/master/pkg/clients/helm.go). When I upgraded Go to 1.14.0 I started to get the following error message.

```
2020-03-01T10:57:43.033Z [ERROR] Unable to apply blueprint: error="Error running chart: template: consul/templates/client-daemonset.yaml:121:23: executing "consul/templates/client-daemonset.yaml" at <(.Values.client.join) and (gt (len .Values.client.join) 0)>: can't give argument to non-function .Values.client.join"
```

I have tried to debug this but currently stumped, downgrading to Go 1.13.8 fixes the issue. Wondering if this is something to do with Go mod changes?

```
go version go1.13.8 linux/amd64
```

--- a/stable/home-assistant/Chart.yaml
+++ b/stable/home-assistant/Chart.yaml
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.107.7
 description: Home Assistant
 name: home-assistant
-version: 0.12.6
+version: 0.12.7
 keywords:
 - home-assistant
 - hass

--- a/stable/home-assistant/templates/appdaemon-ingress.yaml
+++ b/stable/home-assistant/templates/appdaemon-ingress.yaml
@@ -1,4 +1,4 @@
-{{- if (.Values.appdaemon.enabled) and (.Values.appdaemon.ingress.enabled) }}
+{{- if and (.Values.appdaemon.enabled) (.Values.appdaemon.ingress.enabled) }}
 {{- $fullName := include "home-assistant.fullname" . -}}
 {{- $servicePort := .Values.appdaemon.service.port -}}
 {{- $ingressPath := .Values.appdaemon.ingress.path -}}

--- a/stable/home-assistant/templates/configurator-ingress.yaml
+++ b/stable/home-assistant/templates/configurator-ingress.yaml
@@ -1,4 +1,4 @@
-{{- if (.Values.configurator.enabled) and (.Values.configurator.ingress.enabled) }}
+{{- if and (.Values.configurator.enabled) (.Values.configurator.ingress.enabled) }}
 {{- $fullName := include "home-assistant.fullname" . -}}
 {{- $servicePort := .Values.configurator.service.port -}}
 {{- $ingressPath := .Values.configurator.ingress.path -}}

--- a/stable/home-assistant/templates/deployment.yaml
+++ b/stable/home-assistant/templates/deployment.yaml
@@ -170,7 +170,7 @@ spec:
                   name: {{ template "home-assistant.fullname" . }}-configurator
                   key: hass-api-password
             {{- end }}
-            {{- if (.Values.configurator.username) and (.Values.configurator.password) }}
+            {{- if and (.Values.configurator.username) (.Values.configurator.password) }}
             - name: HC_USERNAME
               valueFrom:
                 secretKeyRef:

--- a/stable/home-assistant/templates/vscode-ingress.yaml
+++ b/stable/home-assistant/templates/vscode-ingress.yaml
@@ -1,4 +1,4 @@
-{{- if (.Values.vscode.enabled) and (.Values.vscode.ingress.enabled) }}
+{{- if and (.Values.vscode.enabled) (.Values.vscode.ingress.enabled) }}
 {{- $fullName := include "home-assistant.fullname" . -}}
 {{- $servicePort := .Values.vscode.service.port -}}
 {{- $ingressPath := .Values.vscode.ingress.path -}}

